selfdrive/car/hyundai/hyundaican.py

```diff
@@ -119,6 +121,12 @@ def create_acc_commands(packer, enabled, accel, upper_jerk, idx, lead_visible, s
     "aReqValue": accel,  # stock ramps up and down respecting jerk limit until it reaches aReqRaw
     "CR_VSM_Alive": idx % 0xF,
   }
+
+  # send CF_VSM_ConfMode and AEB_Status to cars that do not accept FCA messages, otherwise TCS13|ACCEnable would fault
```

Comment:
can you make this comment more clear? not sending these signals won't make it fault, sending the FCA messages below would. (this is to show AEB disabled in the dash right?)

Reply:
Here are the scenarios we tested:
1. Not sending either - Fault ❌
2. Send `CF_VSM_ConfMode = 1` only - Fault ❌
3. Send `AEB_Status = 1` only - Fault ❌
4. Send `CF_VSM_ConfMode = 1` and `AEB_Status = 1` - No fault, AEB amber light stays on ✅

Reply:
Hmm, so you weren't able to forward the message through without a fault?